core/src/main/scala/tech/beshu/ror/boot/ReadonlyRest.scala

```diff
@@ -263,21 +266,28 @@ class ReadonlyRest(coreFactory: CoreFactory,
 }
 
 object ReadonlyRest {
-  type AuditSinkCreator = AuditCluster => AuditSinkService
+  trait AuditSinkCreator {
```

Comment:
I wonder if this trait should be defined in the `tech.beshu.ror.accesscontrol.audit` package. WDYT?